push-to-neofs.py

```diff
@@ -148,33 +242,45 @@ def push_files_to_neofs(
     lifetime: int,
     put_timeout: int,
     password: str,
+    replace_object: bool,
 ) -> None:
     if not os.path.exists(directory):
         raise Exception(f"Directory '{directory}' does not exist.")
     if not os.listdir(directory):
         raise Exception(f"Directory '{directory}' is empty.")
 
-    base_cmd = (
-        f"NEOFS_CLI_PASSWORD={password} neofs-cli --rpc-endpoint {endpoint} "
-        f"--wallet {wallet}  object put --cid {cid} --timeout {put_timeout}s"
-    )
     if lifetime is not None and lifetime > 0:
         current_epoch = get_current_epoch(endpoint)
         expiration_epoch = current_epoch + lifetime
-        base_cmd += f" --expire-at {expiration_epoch}"
 
-    base_path = os.path.abspath(directory)
-    for subdir, dirs, files in os.walk(base_path):
-        for filename in files:
-            push_file(
-                base_path,
-                subdir,
-                url_path_prefix,
-                filename,
-                attributes,
-                base_cmd,
-                put_timeout,
+    files = get_file_info(directory, url_path_prefix)
+
+    if replace_object:
+        existing_objects = []
+        for file in files:
+            search_attrs = compile_attributes(
+                file['neofs_path_attr'], file['mime_type'], attributes, output_format="filter_str"
```

Comment:
As far as I understand, Oleg's idea, we need this attribute to be set to get the documents served in a correct way via http gate and view by client's browser.

Reply:
Of course we do. Why do we care about them when we're _deleting_ objects?

Reply:
I use the same function, compile_attributes, for push objects as well. As for deleting, I mean to make a search with all the attributes we'll use for uploading to make sure we're deleting the right object.

Reply:
Suppose I have a `cat.jpeg` with `text/html` mime type uploaded for whatever reason. I want to update it with a new one using `image/jpeg`. It will fail with the current code for no real reason. We select objects via `FilePath` during download, we upload them as files as well. They should be replaced as files, MIME is there and is needed there as additional data for gateway to use and browser to handle properly, but it's not related to object management.

Reply:
Fixed. Searching for FilePath only for deleting objects.